docs/build/guides/archival/create-restoration-footprint-js.mdx

```diff
@@ -4,6 +4,12 @@ hide_table_of_contents: true
 description: Manually set up a restoration footprint with JavaScript SDK
 ---
 
+:::info
+
+The manual operation (`RestoreFootprintOp`) is (for the most part) no longer needed starting in Protocol 23, as archived entries are automatically restored when they appear in a footprint during `InvokeHostFunctionOp`. These examples are left here for historical context.
```

Comment:
I think 'historical context' is a bit misleading as sometimes this operation is useful even with p23. I think we can say that the examples are here to support the rare use cases such as:
- if auto-restoration makes transaction too large to fit into the network limits, the restoration can be performed separately via this op
- if contract developers want to make sure that they pay the restoration fees (as opposed to their users), they can restore their entries on-demand with this op